Assets/Plugins/Source/Editor/Utility/GUIEditorUtility.cs

```diff
@@ -1004,9 +1010,9 @@ private static SetOfNamed<EOSClientCredentials> RenderInput(ConfigFieldAttribute
                 "Enter your client information here as it appears in the Epic Dev Portal.",
                 "https://dev.epicgames.com/docs/dev-portal/product-management#clients",
                 clientCredentialsCopy,
-                (rect, item, nameAsLabel) =>
+                (rect, item, nameAsLabel) => // Things function renders input for a EOSClientCredential item.
                 {
-                    float remainingWidth = rect.width;
+                    float remainingWidth = rect.width - 5f;
```

Comment:
nit: I'm wondering if we want to assign '5' to a variable representing 'X button width' or something. I see '5' used in 4 places in this PR.